css/nav.less

```diff
@@ -488,6 +488,24 @@ nav {
   }
 }
 
+.leftNavIsOpen #topNav .main-menu {
+  @media (max-width: (@main-menu-hide-breakpoint + @left-nav-width - 1)) {
+    display: none;
```

Comment:
I could adjust the widths so that things could still fit, what this would mean is just setting  the break point higher, so not using the extra room.